create_pnor_image.pl

```diff
@@ -229,7 +229,8 @@
 }
 $build_pnor_command .= " --fpartCmd \"fpart\"";
 $build_pnor_command .= " --fcpCmd \"fcp\"";
-print "###############################";
+
+print "Running command build_pnor_command \n";
```

Comment:
is this helpful?  build_pnor_command doesn't have a $ in front of it .. so it will literally print "build_pnor_command" instead of what the command actually ended up being.  generally the run_command already prints the incoming command, so this might be a bit unnecessary

Reply:
I've amended this change to the commit.